### PR TITLE
Clean up magic numbers in windows

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

File: src/openrct2/ride/ride.h [hunk 1]
```diff
@@ -806,7 +806,7 @@ enum {
 	RIDE_CONSTRUCTION_STATE_MAZE_BUILD,
 	RIDE_CONSTRUCTION_STATE_MAZE_MOVE,
 	RIDE_CONSTRUCTION_STATE_MAZE_FILL
-};
+} RIDE_CONSTRUCTION_STATE_IDX;
```
Comment (IntelOrca) [inline src/openrct2/ride/ride.h]: This would be `RIDE_CONSTRUCTION_STATE` to match the enum member prefixes.

**Comment by IntelOrca** [comment]
@marijnvdwerf What is the status of this PR?

**Comment by marijnvdwerf** [comment]
I probably should make `_rideConstructionState` a non-enum again. Or add `default` cases to all switches.

File: src/openrct2/windows/ride_construction.c [hunk 2]
```diff
@@ -87,6 +87,10 @@ enum {
 	WIDX_SEAT_ROTATION_ANGLE_SPINNER_DOWN,
 };
 
+static_assert(WC_RIDE_CONSTRUCTION_WIDX_CONSTRUCT == WIDX_CONSTRUCT, "WIDX doesn\'t match");
```
Comment (Gymnasiast) [inline src/openrct2/windows/ride_construction.c]: Is the backslash necessary?

**Comment by Gymnasiast** [comment]
LGTM code-wise, although I'm not 100% sure about the backslash I left a line note about.

**Comment by IntelOrca** [comment]
`gCurrentToolId` is actually a cursor id. So perhaps use the cursor enum instead of a tool enum?
Its really should be called `gToolCursor` or something, but it is used by the top toobar as a tool identity as well which really should be a separate variable (in case two tools use the same cursor).

**Comment by marijnvdwerf** [comment]
I think tools and cursors should be split up. The fact that the tool determines the cursor seems more like an implementation detail to me.

**Comment by Gymnasiast** [comment]
Since @IntelOrca approved it, could you rebase this @marijnvdwerf ?

**Pull request merged**